<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LHTU05CG84251TZ" name="Member Services Test" tests="1" failures="1" timestamp="5 Apr 2020 19:43:10 GMT" time="0.57" errors="0">
  <testcase name="MemberDetails" time="0.475" classname="com.fp.service.client.testcases.MemberServicesTestSuite">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.fp.service.client.testcases.MemberServicesTestSuite.MemberDetails(MemberServicesTestSuite.java:44)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- MemberDetails -->
</testsuite> <!-- Member Services Test -->
